app/src/main/java/org/literacyapp/chat/ChatActivity.java

```diff
@@ -63,6 +66,29 @@ protected void onStart() {
         arrayAdapter = new MessageListArrayAdapter(getApplicationContext(), textMessages);
         mListPreviousMessages.setAdapter(arrayAdapter);
 
+        mTextMessage.addTextChangedListener(new TextWatcher() {
+            @Override
+            public void beforeTextChanged(CharSequence charSequence, int i, int i1, int i2) {
+
+            }
+
+            @Override
+            public void onTextChanged(CharSequence charSequence, int i, int i1, int i2) {
+
+            }
+
+            @Override
+            public void afterTextChanged(Editable editable) {
+
+                if(editable.toString().trim().length() > 0){
+                    mButtonSend.setColorFilter(Color.GREEN);
+                } else {
+                    mButtonSend.setColorFilter(Color.BLACK);
```

Comment:
@jogrimst with setTint we can modify the color fo vector images, using drawables as in this example and with this method setTintList() we can use our own configuration XML: http://stackoverflow.com/questions/34412742/android-how-to-set-stroke-color-for-vector-drawable-programmatically?answertab=votes#tab-top
However with setColorFilter we just can use colors from the android.graphics library for modify vector images.